newdle/client/src/components/Home.js

```diff
@@ -1,15 +1,74 @@
 import React from 'react';
-import {useSelector} from 'react-redux';
-import {Link} from 'react-router-dom';
-import {isLoggedIn} from '../selectors';
+import {Button} from 'semantic-ui-react';
+import LoginRequired from './LoginRequired';
+import {useRouter} from '../util/router';
+import {Icon} from 'semantic-ui-react';
+
+import {ReactComponent as TeamIcon} from '../images/team.svg';
+import {ReactComponent as TimetableIcon} from '../images/timetable.svg';
+import {ReactComponent as DiscussIcon} from '../images/discuss-issue.svg';
+import {ReactComponent as GitHubIcon} from '../images/github.svg';
+
+import styles from './Home.module.scss';
 
 export default function Home() {
-  const isUserLoggedIn = useSelector(isLoggedIn);
+  const router = useRouter();
 
   return (
-    <>
-      <h2>Home</h2>
-      {isUserLoggedIn && <Link to="/new">Create newdle</Link>}
-    </>
+    <div>
+      <div className={styles.container}>
+        <h2>How does it work?</h2>
+        <ol className={styles.box}>
+          <li>
+            <TeamIcon className={styles.icon} />
+            Choose your participants
+          </li>
+          <li>
+            <TimetableIcon className={styles.icon} />
+            Set the time slots based on their availability
+          </li>
+          <li>
+            <DiscussIcon className={styles.icon} />
+            <strong>newdle</strong> will collect the answers!
+          </li>
+        </ol>
+        <div className={styles.button}>
+          <LoginRequired
+            component={Button}
+            color="violet"
+            icon
+            labelPosition="right"
+            size="big"
+            onClick={() => {
+              router.history.push('/new');
+            }}
+          >
+            Get started
+            <Icon name="caret right" />
+          </LoginRequired>
+        </div>
+        <div className={styles.footer}>
+          <p className={styles.opensource}>
+            newdle is Open Source Software
+            <a href="https://github.com/indico/newdle">
+              <GitHubIcon />
+            </a>
+          </p>
+          <p className={styles.cern}>
+            Made at <a href="https://home.cern">CERN</a>, the place where the web was born.
+          </p>
+          <p className={styles.credits}>
+            &quot;Team&quot;, &quot;Timetable&quot; and &quot;Discuss Issue&quot; icons made by{' '}
```

Comment:
can't we use normal `"` quotes here? and tbh i'd look for icons where no attribution is necessary...

Reply:
I get a warning from ESLint React plugin. We could disable the error, but I wonder if it's not just easier to comply.
![image](https://user-images.githubusercontent.com/2699/65407105-becd0c80-dde1-11e9-8146-671b8ce22cd0.png)

Reply:
As for the icons... I guess we could use FontAwesome, but the icons are not as cool.

Reply:
ok that rule is weird... it looks like it has a setting where you an specify what to forbid (e.g. only `>`)

Reply:
Fixed.